pkg/tools/tools.go

```diff
@@ -192,7 +192,8 @@ func parseNSUrl(urlString string) (*NSUrl, error) {
 		}
 		result.Intf = path[1]
 	}
-	result.NsName = path[0]
+	// Remove possible leading spaces from network service name
+	result.NsName = strings.Trim(path[0], " ")
```

Comment:
Might it not make more sense to call [strings.TrimSpace(urlString)](https://golang.org/pkg/strings/#TrimSpace) on the incoming argument to parseNSUrl ?

Reply:
done